packages/api-v4/README.md

```diff
@@ -0,0 +1,344 @@
+# Instructions for using the API-V4
+
+Azure Functions v4 is the latest version of the Node.js programming model for Azure Functions. It comes with a bunch of new features and improvements, such as:
+
+* Flexible folder structure
+* Being able to define function.json directly in the function's in the code
+* New HTTP trigger types
+* Improved IntelliSense
+* Timer Trigger (TypeScript)
+* Durable Functions (TypeScript)
+  
+I wrote an article explaining how this whole migration process went and how you can do the same. You can read the article [here](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-guide-migrating-v3-to-v4-programming-model-for/ba-p/3897691).
+
+## How to run the new api-v4?
+
+It's very important that, as soon as you clone this repository, create the `local.settings.json` file and add the following block of code:
+
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "node",
+    "AzureWebJobsFeatureFlags": "EnableWorkerIndexing"
+  }
+}
+```
+
```

Comment:
Can you include the same intruction as for the `packages/api/`?

<pre>

## Steps to start the API

1. fork or clone the repository locally
2. assuming you are in the folder containing your clone, go to the terminal and run

```bash
cd packages/api-v4
```

3. create the `local.settings.json` file and add the following block of code:

```json
{
  "IsEncrypted": false,
  "Values": {
    "FUNCTIONS_WORKER_RUNTIME": "node",
    "AzureWebJobsFeatureFlags": "EnableWorkerIndexing"
  }
}
```

4. Now install all the dependencies for the API and run it

```bash
npm install && npm start
```
</pre>